src/encoding/text.rs

```diff
@@ -616,7 +620,7 @@ mod tests {
     fn encode_counter() {
         let counter: Counter = Counter::default();
         let mut registry = Registry::default();
-        registry.register("my_counter", "My counter", counter.clone());
+        registry.register("my_counter", "My counter", counter);
```

Comment:
What do you think of adding a commit here that alters `.github/workflow` to lint the test code as well?

Reply:
I added `--workspace --all-targets` to the arguments of the clippy action. That should include all packages and include tests.